src/components/Person/Person.jsx

```diff
@@ -1 +1,47 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${partner}`
+          : 'I am not married'
+        }
+      </p>
+    </section>
+  );
+};
+
+Person.propTypes = {
+  person: PropTypes.shape({
```

Comment:
why do you use shape here? )))

Reply:
Because as argument in props we get object, that should have specific values (name, agge, sex, etc)